### PR TITLE
import-module incorrectly reporting a loaded module was not found

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
missing break after module was loaded successfully so loop continues
but module isn't "found" and thus reports error even though loaded

Fix https://github.com/PowerShell/PowerShell/issues/5232

### CODE REVIEW

**Review by daxian-dbw — APPROVED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1 [hunk 1]
```diff
@@ -27,6 +39,11 @@
         { $script:module = Import-Module $moduleName -PassThru } | Should Not Throw
```
Comment (JamesWTruher) [inline test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1]: Please add `-ErrorAction Stop` as a failed load is not a terminating error

**Review by JamesWTruher — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
Shall we add the '[Feature]' tag to run all tests?

File: test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1 (hunk 1 above)
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1]: Will add

**Comment by SteveL-MSFT** [comment]
@daxian-dbw it wouldn't hurt to run [feature] tests, will update

**Review by PaulHigin — APPROVED** [review]
LGTM

**Comment by SteveL-MSFT** [comment]
@daxian-dbw looks like for mac Travis-CI installed beta.8, but the scripts tried to start pwsh.  Did we publish beta.9 to homebrew already?

**Comment by daxian-dbw** [comment]
@SteveL-MSFT The PR to update powershell.beta.9 in homebrew is here: https://github.com/caskroom/homebrew-cask/pull/40043

We submitted the PR early this morning, but it hasn't been merged yet.

**Comment by SteveL-MSFT** [comment]
@daxian-dbw thanks, I guess we'll just wait for that to merged then I'll restart the macOS job

**Comment by adityapatwardhan** [comment]
@SteveL-MSFT restarted the Travis-CI macOS job. The homebrew PR is merged.

**Comment by SteveL-MSFT** [comment]
@JamesWTruher can you re-review?

**Review by JamesWTruher — APPROVED** [review]

**Pull request merged**